gems/pending/spec/openstack/amqp/openstack_rabbit_event_monitor_spec.rb

```diff
@@ -48,4 +46,25 @@
     # get back to this piece
   end
 
+  context "remove legacy queues" do
```

Comment:
Change this to match the method being called.  Since it's private just indicate that: `context "#remove_legacy_queues (private)" do`